src/Aspire.Dashboard/Components/Controls/ResourceDetails.razor

```diff
@@ -121,6 +121,35 @@
                               HighlightText="@_filter"
                               GridTemplateColumns="1fr 1.5fr" />
             </FluentAccordionItem>
+            @if (Resource.IsContainer())
+            {
+                <FluentAccordionItem Heading="@ControlStringsLoc[nameof(ControlsStrings.ResourceDetailsVolumesHeader)]" Expanded="true">
+                    <div slot="end">
+                        <FluentBadge Appearance="Appearance.Neutral" Circular="true">
+                            @FilteredVolumes.Count()
+                        </FluentBadge>
+                    </div>
+                    <PropertyGrid TItem="VolumeViewModel"
+                                  Items="@FilteredVolumes"
+                                  NameColumnValue="(vm) => vm.Source"
+                                  ValueColumnValue="(vm) => vm.Target"
+                                  IsNameSortable="false"
+                                  IsValueSortable="false"
+                                  EnableValueMasking="false"
+                                  HighlightText="@_filter"
+                                  GridTemplateColumns="1fr 1.5fr">
+                        <ExtraValueContent>
+                            <div>
+                                @if (context.IsReadOnly)
+                                {
+                                    <FluentIcon Value="@(new Icons.Regular.Size12.LockClosed())" />
```

Comment:
The icon is a little small. We use 16x16 icons with inline text in other places. For example, the state icon on the resources screen:

https://github.com/dotnet/aspire/blob/b6e3e16393835fd67ca482da440ac7efb158f8c1/src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor#L14-L17

Can you make this 16x16 and make the icon and text vertically middle aligned with each other?